beacon-chain/node/prometheus.go

```diff
@@ -0,0 +1,56 @@
+package node
+
+import (
+	"fmt"
+	"github.com/prometheus/client_golang/prometheus"
+	"os"
+)
+
+type bcnodeCollector struct {
+	DiskBeaconchainBytesTotal *prometheus.Desc
+	dbPath string
+}
+
+func startBeaconNodePromCollector(dbPath string) error {
+	namespace := "bcnode"
+	c := &bcnodeCollector{
+		DiskBeaconchainBytesTotal: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "disk_beaconchain_bytes_total"),
+			"Total hard disk space used by the beaconchain database, in bytes.",
+			nil,
```

Comment:
What are these nils? a small comment would be helpful to a reader `nil, /* some config that we don't need */`

Reply:
those params are optional prometheus labels. I think that it is easy enough to peek at the implementation of `prometheus.NewDesc` and see a description for them (in goland they'll be annotated with parameter names from the receiver method). LMK if you still want me to add comments.